test/suite/test_assert_durable_ts01.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
```

Comment:
@bvpvamsikrishna thanks for writing this test. Please rename it to `test_assert05.py` so that it is clearly related to the other assert tests. All those tests perform tests with timestamps and the `assert` setting. This is exactly in the same area.

Reply:
sure. Thank you.